docs/modules/trino/pages/usage_guide/catalogs/generic.adoc

```diff
@@ -0,0 +1,55 @@
+= Generic
+
+As Trino supports a lots of different connectors, we can't possible cover all the available connectors.
+In case the trino-operator does not support a specific connector you can use this connector to pull in any connector Trino supports.
+You are one you own figuring out how to configure and use this connector and we can't possible support all the catalogs available.
```

Comment:
I dont like this. I would point to the Trino docs and e.g. mention that the connector name must match the one in the docs and that properties can be put here etc...
```suggestion
You are on your own figuring out how to configure and use this connector and we can't possible support all the catalogs available.
```